api/src/main/java/edu/cornell/library/scholars/webapp/controller/api/distribute/rdf/graphbuilder/ParallelGraphBuilder.java

```diff
@@ -0,0 +1,14 @@
+/* $This file is distributed under the terms of the license in /doc/license.txt$ */
+
+package edu.cornell.library.scholars.webapp.controller.api.distribute.rdf.graphbuilder;
+
+/**
+ * TODO
+ * 
+ * Iterate like the drill-down builder (share code?) Each sub-request is handled
+ * by a thread-pool, whose size can be configured. Requires the use of
+ * ThreadsafeGraphBuilders.
+ */
+public class ParallelGraphBuilder {
```

Comment:
We should remove this class or add implementation into this class

Reply:
Removed